static/js/index.js

```diff
@@ -385,22 +385,26 @@ function stringifyRooms(arr) {
 function renderPeriod(periodObj) {
   var popupContainer = document.getElementById("popupContainer");
 
+  // Gets the CSS style
+  var style = getComputedStyle(document.body);
+  var bgColor = style.getPropertyValue('--background')
+  innerHTMLStyle = "style='background-color: " + bgColor.toString() + "'";
   popupContainer.innerHTML =
     '<div class="period" layout vertical>' +
```

Comment:
can you just apply the style to the parent div here and have it cascade to its children, e.g. 
```
.period > * {
  background-color: var(--background)
}
```

Reply:
Unfortunately, because of the way we are loading in this HTML content through JavaScript, the CSS rules don't apply for some reason.